cli/cmd/admin/start.go

```diff
@@ -61,6 +62,7 @@ type Config struct {
 	EmailSenderEmail       string                 `split_words:"true"`
 	EmailSenderName        string                 `split_words:"true"`
 	EmailBCC               string                 `split_words:"true"`
+	RedisAddr              string                 `default:"" split_words:"true"`
```

Comment:
I would propose renaming it to `RedisURL` and then using `redis.ParseURL` to parse it into a Redis options object. This enables setting passwords and other config as well without.

Reply:
Done